test/shared/src/main/scala/zio/test/Assertion.scala

```diff
@@ -473,4 +473,52 @@ object Assertion {
         case t: Throwable => assertion(t)
       }
     }
+
+  final def isEmptyString: Assertion[String] = Assertion.assertion(s"isEmpty")() { actual =>
+    actual.isEmpty
+  }
+
+  final def isNonEmptyString: Assertion[String] = Assertion.assertion(s"isNonEmptyString")() { actual =>
+    actual.nonEmpty
+  }
+
+  final def equalsIgnoreCase(other: String): Assertion[String] = Assertion.assertion("equalsIgnoreCase")(param(other)) {
+    actual =>
+      actual.equalsIgnoreCase(other)
+  }
+
+  final def startsWith(prefix: String): Assertion[String] = Assertion.assertion("startsWith")(param(prefix)) { actual =>
+    actual.startsWith(prefix)
+  }
+
+  final def endsWith(suffix: String): Assertion[String] = Assertion.assertion("endsWith")(param(suffix)) { actual =>
+    actual.endsWith(suffix)
+  }
+
+  final def matchesRegex(regex: String): Assertion[String] = Assertion.assertion("matchesRegex")(param(regex)) {
+    actual =>
+      actual.matches(regex)
+  }
+
+  final def approximatelyEquals[A](range: NumericRange[A]): Assertion[A] =
+    Assertion.assertion("approximatelyEquals")(param(range)) { actual =>
+      range.isInRange(actual)
+    }
+
+  final def isEmpty: Assertion[Iterable[_]] = Assertion.assertion("isEmpty")() { actual =>
+    actual.isEmpty
+  }
+
+  final def isNonEmpty: Assertion[Iterable[_]] = Assertion.assertion("isNonEmpty")() { actual =>
+    actual.nonEmpty
+  }
+
+  final def containsElement[A](element: A): Assertion[Iterable[A]] =
```

Comment:
How is this different from the existing `contains` assertion?

Reply:
it's no different ... missed we already had this.